object-cache.php

```diff
@@ -1258,6 +1261,7 @@ public function build_client_parameters( $redis_server ) {
 					// @phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
 					'auth' => ! empty( $_SERVER['CACHE_PASSWORD'] ) ? $_SERVER['CACHE_PASSWORD'] : null,
 					'database' => ! empty( $_SERVER['CACHE_DB'] ) ? intval( $_SERVER['CACHE_DB'] ) : $database,
+					'timeout' => isset( $_SERVER['CACHE_TIMEOUT'] ) ? intval( $_SERVER['CACHE_TIMEOUT'] ) : $timeout,
```

Comment:
This check's current position in the Pantheon config conditional is a problem, as it only would get used if config set by the Pantheon env exists (CACHE_HOST et.al). Since this is not a config value Pantheon envs set, better to move it outside of the current block (probably up at 1250?) making this more portable.